Contents/Resources/Series/Shoko Series Scanner.py

```diff
@@ -120,15 +122,16 @@ def Scan(path, files, mediaList, subdirs, language=None, root=None):
                 seasonStr = try_get(episode_data, 'season', None)
                 if episode_data['eptype'] == 'Credits': seasonNumber = -1 #season -1 for OP/ED
                 elif episode_data['eptype'] == 'Trailer': seasonNumber = -2 #season -2 for Trailer
-                elif seasonStr == None:
+                elif Prefs['SingleSeasonOrdering'] or seasonStr == None:
```

Comment:
I didn't understand.. Specials would still be Season 0 right?

Reply:
With or without that preference, if AniDB has an episode listed as a "Special" in the main series, it belongs to the same series. OVAs on the other hand are a different story because they have different entries on AniDB. Their eptype would is listed as "Episode", so they'll be Season 1 of that separate OVA series.

Reply:
Can you show a screenshot of an example making sure that's the case?

Reply:
![Screenshot_20200509_194614](https://user-images.githubusercontent.com/26010946/81476396-62fbaa80-922f-11ea-8faa-1ab8e38fe7e4.png)
![Screenshot_20200509_195745](https://user-images.githubusercontent.com/26010946/81476401-6727c800-922f-11ea-8c04-6725f510eac0.png)

Angel Beats specials are listed as "Specials" in the main series, so season 0.
Another's OVA is a different series on AniDB, so here too.